samples/server/petstore/java-play-framework-fake-endpoints/app/controllers/FakeApiController.java

```diff
@@ -68,12 +70,13 @@ public Result fakeOuterCompositeSerialize() throws Exception {
         } else {
             body = null;
         }
+
         OuterComposite obj = imp.fakeOuterCompositeSerialize(body);
         if (configuration.getBoolean("useOutputBeanValidation")) {
             SwaggerUtils.validate(obj);
         }
-        JsonNode result = mapper.valueToTree(obj);
-        return ok(result);
+            JsonNode result = mapper.valueToTree(obj);
```

Comment:
I could continue on and on but like I said earlier, just make sure there is no changes in the previous samples and I will approve this PR. If you have trouble using the mustache file (it can be tricky), let me know exactly where you are struggling and I will try to help.

Reply:
there is an easy way to fix the tabulation issue without having to add each one of them manually? :cold_sweat:

Reply:
Well it is generated, so if you fix the mustache file, it will be fix everywhere. Never edit manually the files in the "sample" folder.

Reply:
No, I meant in the mustache file

Reply:
I think you only need to change it at one place. I don't have time right now to check it out but I'm sure you'll figure it out by trial and error :) If you are completely clueless, I could check it next week.

Reply:
ok I fixed the other scenarios, but the async one still lacks tabulations :confused: